### PR TITLE
fix(VRating): make icons tab-accessible

### LINKED ISSUE #10820: [Bug Report] v-rating does not allow accessible keyboard navigation
### Environment
**Vuetify Version:** 2.2.11
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.132, Mozilla Firefox
**OS:** Windows 10

### Steps to reproduce
- Turn NVDA or JAWS on 
- Attempt to use the keyboard to navigate (tab) to the stars and select (enter) a rating. 

### Expected Behavior
- A user should be able to tab to, focus on, and select/unselect each star using only the keyboard
- The same CSS applied on :focus should also be as similar as possible to :hover
- When tabbing through each star, the screen reader should read the total number of stars, which star they are on, whether or not it is selected, and be notified when a state changes. 

### Actual Behavior
The stars are completely inaccessible to the keyboard and doesn't announce in screen readers.

### Reproduction Link
<a href="https://codepen.io/flpehr0/pen/bGdvbbX" target="_blank">https://codepen.io/flpehr0/pen/bGdvbbX</a>

### Other comments
Possible solutions include: 
- use of aria role="group" to associate them as a set
- use of aria labels on the stars to label where they are in the set
- using tabindex to allow keyboard focus
- using aria-current or aria-pressed to show selection
- expose the code for the stars to the developer so we can add custom accessibility attributes. 

Resource for aria-current and other aria tags: https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/current/

### PR DESCRIPTION
**Pull request by rkrasnan**
## Description

Set tabindex to 0 so we can tab to the button and press enter.
resolves #10820 

## Motivation and Context

This makes the buttons generated by v-rating accessible.
fixes #10820 

## How Has This Been Tested?

Tested visually. Ran with NVDA, can tab to the buttons and aria-labels are read.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by rkrasnan** [comment]
Please let me know if there is something more that needs to be done!

File: packages/vuetify/src/components/VRating/VRating.ts [hunk 1]
```diff
@@ -224,7 +224,7 @@ export default mixins(
 
       return this.$createElement(VIcon, this.setTextColor(this.getColor(props), {
         attrs: {
-          tabindex: -1,
+          tabindex: 0,
```
Comment (KaelWD) [inline packages/vuetify/src/components/VRating/VRating.ts:227]: I think you can just remove this line? VIcon renders a button when it has a click listener so it should be focusable by default.

**Comment by KaelWD** [comment]
Thanks, not sure how that was missed for so long, even in #12345

File: packages/vuetify/src/components/VRating/VRating.ts (hunk 1 above)
Comment (rkrasnan) [reply packages/vuetify/src/components/VRating/VRating.ts]: great! will do

**Comment by rkrasnan** [comment]
Happy to help! I removed the tabindex completely. Let me know if anything else is needed. Thanks!

**Comment by rkrasnan** [comment]
It works for me in the vercel preview! It might need some aria-selected logic added as well, but that's a different issue.

**Comment by rkrasnan** [comment]
hey hey Kael, anything else needed from me (other then patience)? I'd love to get this fix into my website.

**Review by KaelWD — APPROVED** [review]

**Pull request merged**